policies/election-procedures.md

```diff
@@ -0,0 +1,164 @@
+   - Post the ballot measures in the OpenAstronomy Discourse voting-members.
+   - One post per ballot measure
+5. Set up the election on Helios Voting (see next section)
+
+## Running an election with Helios
+
+1. Log into [helios](https://vote.heliosvoting.org/)
+2. Click “Create an election”
+   - Short name convention: astropy-<description>-<month>-<year> 
+     For example: astropy-voting-members-july-2022
+   - Name convention: Astropy <description> — <month> <year> 
+     For example: Astropy Voting Members — July 2022
+   - Description: Write a one sentence description of the election.
+   - Check “Private?” to ensure that only registered voters can see the ballot.
+   - Voting starts at: select the predetermined date for the election to start
+     (12:00 UTC for the time)
+   - Voting ends at: two weeks after the start date
+   - All other field left as default values
+3. Now we have to add questions to the ballot. Click the “questions(0)” link to add a new question.
+   - Add one question for each nominee: “Do you support adding <NAME> as an Astropy voting member?”
+     - Make each required by selecting “select between 1 and 1 answers”
+     - Add “Yes”, “No”, “Abstain” as possible answers
+   - Add one question for each ballot measure
+     - Make each required by selecting “select between 1 and 1 answers”
+     - Add “Yes”, “No”, “Abstain” as possible answers
+4. Make sure the list of current voting members is complete
+   - The list is in [google sheets](https://docs.google.com/spreadsheets/d/18VOl-EAwefcwi1aNLWXnZ0-a9WTBx4ry_QB8t23tYKw/edit#gid=0)
+   - Download the list as a CSV, remove the first row (the headers), and upload as registered voters
+5. Freeze the ballot and set it to open voting at the start time
+   - Go to the “voters & ballots” link and prepare / send an email with voting
+     information to all voters
+   - DO NOT copy paste to fill the “Body” section of the form, instead use that space to add
+     any additional info you want to send to all voters.
+6. After the ballot closes, “compute encrypted tally” and then “compute results”
+   - Send the results email
+7. Create a PR to add any newly-added voting members to https://github.com/astropy/astropy.github.com
+   (see https://github.com/astropy/astropy.github.com/pull/496 for an example)
+
+
+
+## Email templates
+
+**Note:** any text in “<>” should be updated for future emails.
+
+### Election announcement
+
+Subject: Announcing Astropy Voting Member Election: <July 2022>
+
+Dear Astropy Community,
+
+The next Astropy Voting Member election will be on <July 8, 2022.>
+
+We have already received some nominations from the Roles survey, but are accepting additional
+nominations for new voting members until the end of <June 21> (anywhere on Earth).
+To nominate someone (or yourself) to be a voting member, please fill out this form:
+https://forms.gle/f1hi6MNEmw1RzYPY7 
+
+For the Coordination Committee,
+<sender name>
+
+—------------------
```

Comment:
Not sure what this divider is supposed to do.

Reply:
It's between the templates.... I just copied it over from the google doc, have you got a better suggestion?

Reply:
Does HTML tag work?

```html
<hr/>
```

Would look like this:

<hr/>